### ISSUE TITLE
BUG - LEDs Flickering After Most Recent Update

### ISSUE THREAD

**Issue by Y4m4k** [Artemis-RGB/Artemis#664]
I have LEDs connected to a Razer Chroma Addressable RGB device. After the update, the LEDs continually flicker. It was working fine prior to the update.

UPDATE 1 - when I disable/close Razer Central, the flickering stops. Seems like there is a conflict between Artemis and Razer Central.

UPDATE 2 - re-installed RAZER and issue is now resolved.

**Comment by Y4m4k** [comment]
Spoke to early, I opened Razer and went into Studio... started happening again.

**Comment by diogotr7** [comment]
Are you using my chroma plug-in by any chance?

**Comment by Y4m4k** [comment]
Nope, I dont think I am using any plugins (how can I double check)? I just installed Artemis a couple months ago and was working without any issues, even with Razer installed. I had my keyboard, mouse pad and mouse running Razer RGB and all my internal PC components with Artemis. Now, Ive had to uninstall Razer so that the flickering stops.

**Comment by diogotr7** [comment]
I encountered this issue in Aurora and I've seen user reports online about Overwatch causing this flickering as well, so I think it's a Razer thing.

**Comment by RobertBeekman** [comment]
I moved this to the plugins repo but I was wondering if this is still an issue?
